### ISSUE TITLE
new-app-release issue

### ISSUE THREAD

**Issue by Elhor02** [obytes/react-native-template-obytes#337]
When I run app-release.yml in actions Tab manually, I expect new-release (on push tags *) to be triggered automatically.
But It is not, the only way for that pipeline to be triggered is to push Tags from local?
Is there any suggestions or modifications I can make, Thank you!!

**Comment by yjose** [comment]
Not really, the action should push the tag to GitHub automatically, At least this is what we are doing, and it's working as expected. I am not sure what caused the issue, but probably it can be a permission issue.

**Comment by Elhor02** [comment]
So my issue was that I should add GH_TOKEN, I know it was mentionned in the documentation.
But I learned that the github bot is not able to trigger workflows, that's why I should have create my PAT (Personal access token)
I hope that it can help anybody that faces the same issue.
